tests/sentry/users/api/endpoints/test_user_details.py

```diff
@@ -120,7 +119,6 @@ def test_simple(self) -> None:
                 "clock24Hours": True,
                 "extra": True,
                 "prefersIssueDetailsStreamlinedUI": True,
-                "prefersChonkUI": True,
```

Comment:
This is the only frontend change which will be removed once the dependent PR is merged